### DISCUSSION TITLE
The DIV area blocks user input as shown by Inspect Element

### DISCUSSION THREAD

**Discussion by James-buzz** [spatie/laravel-support-bubble#13] [Bugs] [answered]
Hello,

As you can see from the image below, the support bubble blocks the content behind from being clickable. 
Is this a design problem or a fault on my end?

![image](https://user-images.githubusercontent.com/16080646/132208106-8d7017e5-e750-4916-bd20-432b84c5389b.png)

**Comment by freekmurze** [comment]
Could you take a look at this @AlexVanderbist

**Comment by AlexVanderbist** [comment, marked as answer]
Tagging a fix for this in a couple of minutes! thanks for pointing this out :)

**Reply by James-buzz** [reply]
No worries!

**Reply by villdar** [reply]
Any solution for this problem?

**Reply by AlexVanderbist** [reply]
@villdar should be fixed on the latest versions... Feel free to open an issues if you're still experiencing this

**Reply by villdar** [reply]
I can't see the tab issues on this rep. But as you can see in this screen the problem stil remain for me. Even if i set the z index of my graph higher.

<img width="1499" alt="Schermata 2021-09-15 alle 18 00 19" src="https://user-images.githubusercontent.com/61460340/133468382-d50da6fa-e8c6-4ddf-a20d-1b2c9c435bd2.png">

My spatie/support-bubble current version is 1.2

**Reply by AlexVanderbist** [reply]
@villdar can you try publishing (or copying) the default config file again? Some changes to CSS classes were in there that I don't see in your screenshot. Mainly `pointer-events-none` is important here to make be able to touch the underlying elements.
![image](https://user-images.githubusercontent.com/6287961/133584914-35ba8b95-4376-4381-8c50-a84c3b00ee22.png)

**Comment by abishekrsrikaanth** [comment]
@AlexVanderbist @freekmurze The same issue exists on mobile as well and the support bubble does show on mobile. Here is a screenshot from flareapp.io. The support bubble doesn't show on mobile but the div blocks access to all the elements behind it. 

![image](https://user-images.githubusercontent.com/1639302/132250305-dde3c084-3c8d-492f-83ef-64037b026c73.png)

**Comment by colinmackinlay** [comment]
@AlexVanderbist The ability to click through to the underlying window appears to be fixed but when the support bubble form is off canvas, for me it overlaps the scrollbar and blocks clicks on that:
![image](https://user-images.githubusercontent.com/7833362/136293358-391ae8c5-1313-4ded-9e7f-25c90ee6582d.png)

**Comment by colinmackinlay** [comment]
The issue remains but seems to be to do with the margin. If I remove `m-4` from the div then I can click in the scroll bar area and it works - the mouse events pass through the main part of the div so I can use the part indicated green here:
![image](https://user-images.githubusercontent.com/7833362/147491318-896d805d-bce1-4b38-a702-7788bba29233.png)

But if I replace `m-4` with `p-4` it still masks the mouse events from the scroll bar so the red part is masked again. Perhaps this might help work out the solution @AlexVanderbist? I wonder if `pointer-events-none` doesn't apply to the margin or padding?
![image](https://user-images.githubusercontent.com/7833362/147491219-89385471-8c91-4e45-bf65-5d906b097bb5.png)
